src/app/suggestion-notifications/suggestion-notifications.module.ts

```diff
@@ -0,0 +1,86 @@
+import { NgModule } from '@angular/core';
```

Comment:
I see this is where we start having modules/services with the `suggestion-notifications` name.  I still find this name odd, but perhaps it'll make more sense for later PRs?